src/main/scala/edu/berkeley/cs/rise/opaque/Utils.scala

```diff
@@ -822,6 +872,12 @@ object Utils extends Logging {
             tuix.ExprUnion.Literal,
             tuix.Literal.createLiteral(builder, valueOffset))
 
+        case (Decrypt(child, dataType), Seq(childOffset)) =>
```

Comment:
Maybe include a comment stating that this should never be evaluated on the driver?